internal/utils_test.go

```diff
@@ -233,6 +233,123 @@ data:
 	}
 }
 
+func TestGetPolicyTemplateFromPolicyTypeManifest(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	manifestFiles := []types.Manifest{}
+
+	// Test manifest is non-root IAM policy type
+	IamManifestPath := path.Join(tmpDir, "configmapIam.yaml")
+	IamManifestYAML := `
+apiVersion: policy.open-cluster-management.io/v1
+kind: IamPolicy
+metadata:
+  name: policy-limitclusteradmin-example
+spec:
+  severity: medium
+  namespaceSelector:
+    include: ["*"]
+    exclude: ["kube-*", "openshift-*"]
+  remediationAction: enforce
+  maxClusterRoleBindingUsers: 5
+`
+
+	err := ioutil.WriteFile(IamManifestPath, []byte(IamManifestYAML), 0o666)
+	if err != nil {
+		t.Fatalf("Failed to write %s", IamManifestPath)
+	}
+
+	manifestFiles = append(
+		manifestFiles, types.Manifest{Path: IamManifestPath},
+	)
+
+	policysetManifestPath := path.Join(tmpDir, "configmapPolicyset.yaml")
```

Comment:
Could you also not use `configmap` in the file name? It's confusing since it implies the file is a `ConfigMap` object.